frontend/test/e2e.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { BrowserContext, expect, test as baseTest } from "@playwright/test";
+import dappwright, { Dappwright, MetaMaskWallet } from "@tenkeylabs/dappwright";
+
+export const test = baseTest.extend<{
+  context: BrowserContext;
+  wallet: Dappwright;
+}>({
+  context: async ({}, use) => {
+    // Launch context with extension
+    const [wallet, _, context] = await dappwright.bootstrap("", {
+      wallet: "metamask",
+      version: MetaMaskWallet.recommendedVersion,
+      seed: "test test test test test test test test test test test junk", // Hardhat's default https://hardhat.org/hardhat-network/docs/reference#accounts
+      headless: false,
+    });
+
+    // Add Sapphire Localnet as a custom network
+    await wallet.addNetwork({
+      networkName: "Sapphire Localnet",
+      rpc: "http://localhost:8545",
+      chainId: 23293,
+      symbol: "ROSE",
+    });
+
+    await use(context);
+  },
+
+  wallet: async ({ context }, use) => {
+    const metamask = await dappwright.getWallet("metamask", context);
+
+    await use(metamask);
+  },
+});
+
+test.beforeEach(async ({ wallet, page }) => {
+  // Use first account from seed. dAppwright adds two accounts by default.
+  await wallet.switchAccount(1); // Selector queries as a string
+  await page.bringToFront();
+});
+
+// TODO: improve test without reloading wallet page
```

Comment:
Is this related to `wallet.page.reload()` calls below? Any insides on this? Is this a known bug?

Reply:
I couldn't get the playwright/dappwright code to interact with the pop-up window similar to this [person](https://github.com/TenKeyLabs/dappwright/issues/318).

I tried `bringToFront` which didn't work either.